lib/workers/global/config/parse/home-dir.ts

```diff
@@ -0,0 +1,74 @@
+import os from 'os';
+import fs from 'fs-extra';
```

Comment:
Better use our fs proxies?

Reply:
I couldn't get these tests to work any way I tried. Any ideas?

Reply:
I'll try to play with tomorrow.